README_github.md

```diff
@@ -0,0 +1,37 @@
+Scribus
+=======
+
+**Note:** This exists as a mirror of Scribus for the Scribus developer community to collaborate on Scribus independently from the main team who will continue to develop on svn.  You can find the official page for Scribus at http://www.scribus.net
+
+Why Scribus on GitHub?
+----------------------
+- GitHub has become a way to quickly and ingeniously collaborate.
+- Git has also seemed to surpass Subversion.
+- GitHub has an enormous developer pool.
+
+
+Through the initiative of several Scribus community members ``ScribusProject`` came in to existence on GitHub, harnessing its powers for community open-source driven development.
+
+Core Devs
+---------
+We find it important to acknowledge the work of past and present core Scribus team.
+Currently the active core-developers consists of Jean Ghali, Craig Bradney, and  Franz Shmidt.
+
+An open source DTP takes a lot of work; it requires a high level of complexity, knowledge of printing standards, graphic standards  etcetera.  Many thanks go to the ongoing work of these hardcore enthusiasts, who keep Scribus alive and kicking.  We hope that through this effort we can attract a wider pool of folks willing to contribute to Scribus.
+
+Scribus Community Managers on Github
+-------------------------------------
+@malex, @aoloe, @luzpaz, @JLuc, @FirasH
+
+Support and Contact
+-------------------
+Leave a message for us in the issue queues.
+
+
+**Note:** All of here work as volunteers and thus we have limited time and availability.  If you don't hear from us (a slim change but still possible) please check out the:
```

Comment:
a slim chance, not change... but, all in all: no chance that they won't hear from us :-) at least for now...
i would call this section: "the scribus communication channels" or something similar...